test/parallel/test-process-chdir.js

```diff
@@ -42,3 +42,11 @@ const err = {
 };
 common.expectsError(function() { process.chdir({}); }, err);
 common.expectsError(function() { process.chdir(); }, err);
+
+// Check that our built-in methods do not have a prototype/constructor behaviour
+// if they don't need to. This could be tested for any of our C++ methods.
+assert.strictEqual(process.cwd.prototype, undefined);
+assert.throws(() => new process.cwd(), /not a constructor/);
+// Make sure that the above checks verify the right thing, i.e. that we're
+// actually looking at a directly exposed C++ binding method.
+assert.strictEqual(process.cwd.toString(), 'function cwd() { [native code] }');
```

Comment:
Possible engine specific `Function#toString()` result 👆.

Reply:
@jdalton Yeah. Previously, the @nodejs/node-chakracore team made adjustments for engine-specific strings like this in their fork and upstreamed them – however, that project seems dead. :|

Does this actually vary between engines in this case?

Reply:
That being said – this actually tests an engine-specific feature, and I don’t know if e.g. chakracore even provides a shim for this.

Reply:
> Does this actually vary between engines in this case?

It has in the past, though there was some spec changes to help normalize things like function name, newlines, etc.

Since you're testing that something throws when you `new` it or `Reflect.construct()` it you could just test that it throws a `TypeError` and call it good.

I think the `toString()` check is extraneous.

Reply:
fwiw the Function.prototype.toString revision recently landed in ecma-262, and all implementations (not including XS) are compliant.

note that the given assertion doesn't actually match match the spec, where any amount of whitespace is allowed between the tokens. (and including the name isn't required)

i'd agree with jdd that it should be removed.